lib/_tls_wrap.js

```diff
@@ -836,10 +836,13 @@ exports.connect = function(/* [port, host], options, cb */) {
 
   var defaults = {
     rejectUnauthorized: '0' !== process.env.NODE_TLS_REJECT_UNAUTHORIZED,
-    ciphers: tls.DEFAULT_CIPHERS,
     checkServerIdentity: tls.checkServerIdentity
   };
 
+  if (tls.DEFAULT_CIPHERS != tls.getLegacyCiphers('v0.10.38')) {
```

Comment:
If one passes `--cipher-list='ECDHE-RSA-AES128-SHA256:AES128-GCM-SHA256:RC4:HIGH:!MD5:!aNULL:!EDH'`, my understanding is that `tls.DEFAULT_CIPHERS` will === `tls.getLegacyCiphers('v0.10.38'))`, and `default.ciphers` won't be set. I haven't tested it, so it may be a wrong assumption.

So I guess that here what is meant is more `if (user didn't pass --enable-legacy-cipher-list=v0.10.38) { ... }`, which is not equivalent to `if (tls.DEFAULT_CIPHERS != tls.getLegacyCiphers('v0.10.38'))`.

Thoughts?